composer.json

```diff
@@ -39,11 +39,14 @@
         "phpunit/phpunit": "^10.4.2",
         "symfony/browser-kit": "^6.3.2|^7.0",
         "symfony/css-selector": "^6.3.2|^7.0",
-        "symfony/debug-pack": "^1.0.10",
+        "symfony/debug-bundle": "^6.3|^7.0",
```

Comment:
FYI, just running `composer install` made these changes.

Reply:
Not a prob in dev, it's actually better if the pack is unpacked for future reliability.